ios/Classes/SocketManagerProxy.m

```diff
@@ -31,7 +31,9 @@ - (instancetype)initWithContext:(id<TiEvaluator>)context url:(NSURL *)url option
   self.timeout = [TiUtils doubleValue:@"timeout" properties:options def:20000.f];
 
   NSDictionary *nativeOptions = [self convertOptions:options];
-  NSLog(@"lookup manager with options %@", nativeOptions);
+#if DEBUG
+  DebugLog(@"lookup manager with options %@", nativeOptions);
+#endif
```

Comment:
Could you please remove the debug log entirely? It's was pushed by accident and there is currently and [issue](https://jira.appcelerator.org/browse/TIMOB-26861) with {{DebugLog}} anyway. Thanks!